### DISCUSSION TITLE
Spotube on Play Store :partying_face:

### DISCUSSION THREAD

**Discussion by KRTirtho** [KRTirtho/spotube#568] [Announcements]
After 2 years of development, I'm happy to announce that,
Spotube is now available on Play Store

All of this was possible only because of your support and encouragement that helped us grow. And now finally, Spotube is officially on Play store to ease the installation of end user and get latest updates more easily

Thanks to all the supporters, financial & code contributors. Spotube is an shinning example of a community working together

Play store link: https://play.google.com/store/apps/details?id=oss.krtirtho.spotube

**Comment by giftedl** [comment]
Are you worried that Google might take it down because of its ad free function?

**Reply by KRTirtho** [reply]
Not worried about the ad free thing. If I pushed ads it'd been worse. As long as I don't make profit out of it, it's fine ig
If I didn't upload it in the playstore, copycats will be a big problem. So let's just hope for the best
